pkg/migrate/migrate.go

```diff
@@ -30,7 +31,7 @@ type Driver interface {
 	Version() (string, error)
```

Comment:
Can you also add context to the `Version()` call as well?

Reply:
Done !